runtime-sdk/src/modules/core/mod.rs

```diff
@@ -98,6 +98,10 @@ pub enum Error {
     #[error("{0}")]
     #[sdk_error(transparent, abort)]
     Abort(#[source] dispatcher::Error),
+
+    #[error("authenticate_tx not handled")]
+    #[sdk_error(code = 19)]
+    AuthenticateTxNotHandled,
```

Comment:
Maybe rename this to something like "no module could authenticate the transaction", `NotAuthenticated`.